internal/cli/atlas/clusters/create.go

```diff
@@ -154,7 +154,7 @@ func (opts *CreateOpts) providerName() string {
 
 func (opts *CreateOpts) newReplicationSpec() atlas.ReplicationSpec {
 	var (
-		readOnlyNodes int64 = 0
+		readOnlyNodes int64
```

Comment:
default is zero

Reply:
ah yes, I review similar change last week -> `var slaveDelay float64 = 0`. I'll keep it in mind going forward.

Reply:
this is just the new linter is picky about it, I would not mind